engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -2405,6 +2516,46 @@ private LogicalOperatorInfo fillLogicalOperatorInfo(OperatorMeta operator)
     return loi;
   }
 
+  private LogicalModuleInfo fillLogicalModuleInfo(ModuleMeta module, boolean flatten)
+  {
+    LogicalModuleInfo lmi = new LogicalModuleInfo();
+    lmi.name = module.getName();
+    lmi.className = module.getModule().getClass().getName();
+    if (flatten) {
+      for (OperatorMeta operatorMeta : getLogicalPlan().getAllOperators()) {
+        if (operatorMeta.getModuleName() != null
+            && (operatorMeta.getModuleName().equals(module.getName()) || operatorMeta.getModuleName().equals(
+                module.getParentModuleName() + "_" + module.getName()))) {
+          lmi.operators.add(fillLogicalOperatorInfo(operatorMeta));
+        }
+      }
+      for (StreamMeta streamMeta : getLogicalPlan().getAllStreams()) {
```

Comment:
You are right. I should have added streamMeta into lmi using operatorMeta.getOutputStreams and operatorMeta.getInputStreams.  Did not notice it then.